doc/content/liq/source-cue.liq

```diff
@@ -0,0 +1,26 @@
+#!./liquidsoap
+
+radio = insert_metadata(sine())
+f = radio.insert_metadata
+
+# BEGIN
+radio =
+  source.cue(
+    title=
+      "My stream",
+    file="backup.mp3",
+    "/tmp/backup.cue",
+    radio
+  )
+output.file(%mp3, "/tmp/backup.mp3", radio)
+
+# END
+thread.run(
+  delay=1.,
+  {f([("artist", "artist1"), ("album", "album1"), ("title", "title1")])}
+)
+thread.run(
+  delay=2.,
+  {f([("artist", "artist2"), ("album", "album1"), ("title", "title2")])}
+)
```

Comment:
Can't we just have a playlit for this example? This is script is more suited for a test than doc..

Reply:
The part which is included in the doc is only the one between `BEGIN` and `END`! This part is for automated testing of the script from the doc!

Reply:
Ha ok!

Reply:
Although CUE were invented for CDs, and everything above 90 minutes will give problems with a real CD player (used for "hidden" tracks before first track), we _should_ allow the minutes field to grow past 99, i.e., allow 3 or 4 digits. A recording of a DJ set can easily be 120/240 or even more minutes.

This might compromise the (non-exisiting, de-facto) standard but that much can’t be burned to a CD anyway, and other software (audio players) can usually (but not always) handle minutes above 90.

It’s unclear if leading zeroes for the 3 2-digit fields `MM:SS:FF` are mandatory, but I’d advise to use a leading zero for values < 10. The `MM` part could simply "overflow" to the left, i.e. something like `123:45:67`.

Reply:
Yes, that's the way it should be right now: `123:02:65`. Thanks for looking this up!